### PR TITLE
implemented Mpz class (gmp package).

### PR DESCRIPTION
**Pull request by tetsurom**
implemented Mpz class (gmp package).

### CODE REVIEW

File: package/gmp/gmp_glue.c [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+/*
+NOTE: mpz_t, mpf_t and mpq_t are pointer.
+*/
+
+typedef struct _kMpz {
+	KonohaObjectHeader h;
+	mpz_t mpz;
+} kMpz;
+
+//typedef struct _kMpf {
```
Comment (imasahiro) [inline package/gmp/gmp_glue.c]: You do not need to define kMpq, kMpf.
Please remove this.

File: package/gmp/gmp_glue.c [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+
+static KMETHOD Mpz_toInt(KonohaContext *kctx, KonohaStack *sfp)
+{
+	kMpz *self = (kMpz*)sfp[0].asObject;
+	RETURNi_(mpz_get_si(self->mpz));
+}
+
+static KMETHOD Int_toMpz(KonohaContext *kctx, KonohaStack *sfp)
+{
+	kMpz *ret = (kMpz*)KLIB new_kObject(kctx, CT_Mpz, 0);
```
Comment (imasahiro) [inline package/gmp/gmp_glue.c]: Using O_ct(sfp[K_RTNIDX].o) is better.

File: package/gmp/gmp_glue.c [hunk 3]
```diff
@@ -0,0 +1,608 @@
...
+	KLIB kNameSpace_loadMethodData(kctx, ns, MethodData);
+/*
+	KDEFINE_INT_CONST IntData[] = {
+			{_KVi(MECAB_NOR_NODE)},
+			{_KVi(MECAB_UNK_NODE)},
+			{_KVi(MECAB_BOS_NODE)},
+			{_KVi(MECAB_EOS_NODE)},
+			{}
+	};
+	KLIB kNameSpace_loadConstData(kctx, ns, KonohaConst_(IntData), pline);
```
Comment (imasahiro) [inline package/gmp/gmp_glue.c]: please remove this.

File: package/gmp/gmp_glue.c (hunk 2 above)
Comment (tetsurom) [inline package/gmp/gmp_glue.c]: Thank you very much. I could not find how to get return type.

**Pull request merged**